h/static/scripts/reducers/annotations.js

```diff
@@ -105,7 +108,8 @@ var update = {
           updatedTags[existing.$$tag] = true;
         }
       } else {
-        added.push(initializeAnnot(annot));
+        added.push(initializeAnnot(annot, 't' + nextTag));
```

Comment:
In master, annotations on both sides of the channel are given local tags generated with `btoa(Math.random())`. The important thing about the tags is that they may be exposed to third-party JS, so shouldn't be connected to the original annotation ID at all.

Here I've gone for the simpler option of using `t1`,`t2` ... `tN` for annotations loaded via the API where `N` is just a serial number that is incremented every time a new annotation is loaded via the API, on the basis that 3rd-party JS code in the page could observe the order in which annotations are loaded if it wanted to, so there isn't much point trying to hide that.

Annotations originating from the page still use the old scheme.